gobiko/apns/client.py

```diff
@@ -213,16 +208,11 @@ def _send_push_request(self, connection, registration_id, json_data, request_hea
 
         if response.status != APNSResponse.Success:
             body = json.loads(response.read().decode('utf-8'))
-            reason = body["reason"] if "reason" in body else None
+            reason = body.get("reason")
 
             if reason:
                 exceptions_module = importlib.import_module("gobiko.apns.exceptions")
-                ExceptionClass = None
-                try:
-                    ExceptionClass = getattr(exceptions_module, reason)
-                except AttributeError:
```

Comment:
No need to catch `AttributeError` to provide a default value,  because [getattr](https://docs.python.org/3/library/functions.html#getattr) has an optional third argument for the default value